src/screens/catalog.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { parse } from 'qs';
+import { Article, Heading, Anchor, Section } from 'grommet';
+
+import FilterComponent from '../components/filter';
+import Tiles from '../components/tiles';
+import CatalogPageTile from '../components/tiles/catalog-page_tile';
+
+import { loadProducts } from '../ducks/products';
+
+class CatalogPage extends React.Component {
+  constructor(props) {
+    super(props);
+    const { location: { search } } = props;
+    this.state = { values: parse(search.substring(1)) };
+  }
+
+  componentDidMount() {
+    this.props.loadProducts(this.state.values);
+  }
+
+  render() {
+    const { filterOnDisplay, products, match: { params } } = this.props;
+
+    let filter;
```

Comment:
maybe it's just me, but if a condition for displaying a component is simple (like in this case), I prefer using it in render() like so: {filterOnDisplay && <FilterComponent/>}

Reply:
just I saw it in Grommet demoApp, but I agree that {filterOnDisplay && } look cleaner